docs/howto/mirroring.md

```diff
@@ -0,0 +1,270 @@
+            "fs:CreateCommit",
+            "fs:CreateMetaRange",
+            "fs:ReadCommit",
+            "fs:ListCommits",
+            "fs:CreateBranch",
+            "fs:DeleteBranch",
+            "fs:RevertBranch",
+            "fs:ReadBranch",
+            "fs:ListBranches"
+         ],
+         "effect": "allow",
+         "resource": "*"
+      }
+   ]
+}
+```
+
+**Alternatively**, we can create a policy with a narrower scope, only for a specific repository and/or mirror:
+
+```json
+{
+   "id": "ReplicationPolicy",
+   "statement": [
+      {
+         "action": [
+            "fs:ListRepositories"
+         ],
+         "effect": "allow",
+         "resource": "*"
+      },
+      {
+         "action": [
+            "fs:ReadRepository",
+            "fs:ReadObject",
+            "fs:ListObjects",
+            "fs:ReadCommit",
+            "fs:ListCommits",
+            "fs:ReadBranch",
+            "fs:ListBranches"
+         ],
+         "effect": "allow",
+         "resource": "arn:lakefs:fs:::repository/{sourceRepositoryId}"
+      },
+      {
+         "action": [
+            "fs:ReadRepository",
+            "fs:CreateRepository",
+            "fs:UpdateRepository",
+            "fs:DeleteRepository",
+            "fs:AttachStorageNamespace",
+            "fs:ReadObject",
+            "fs:WriteObject",
+            "fs:DeleteObject",
+            "fs:ListObjects",
+            "fs:CreateCommit",
+            "fs:CreateMetaRange",
+            "fs:ReadCommit",
+            "fs:ListCommits",
+            "fs:CreateBranch",
+            "fs:DeleteBranch",
+            "fs:RevertBranch",
+            "fs:ReadBranch",
+            "fs:ListBranches"
+         ],
+         "effect": "allow",
+         "resource": "arn:lakefs:fs:::repository/{mirrorId}"
+      },
+      {
+         "action": [
+            "fs:AttachStorageNamespace"
+         ],
+         "effect": "allow",
+         "resource": "arn:lakefs:fs:::namespace/{DestinationStorageNamespace}"
+      }
+   ]
+}
+```
+
+Once a user has been created and the replication policy attached to it, create an access key and secret to be used by the mirroring process.
+
+
+### Authorizing the lakeFS Mirror process to use the replication user
+
+Please [contact Treeverse customer success](mailto:support@treeverse.io?subject=Setting up replication) to connect the newly created user with the mirroring process  
+
+
+### Configuring repository replication
+
+Replication has a stand-alone HTTP API. In this example, we'll use cURL, but feel free to use any HTTP client or library:
+
+```bashhttps://treeverse.us-east-1
+curl --location 'https://<ORGANIZATION_ID>.<SOURCE_REGION>.lakefscloud.io/service/replication/v1/repositories/<SOURCE_REPO>/mirrors' \
+--header 'Content-Type: application/json' \
+-u <ACCESS_KEY_ID>:<SECRET_ACCESS_KEY> \
+-X POST \
+--data '{
+    "name": "<MIRROR_NAME>",
+    "region": "<MIRROR_REGION>",
+    "storage_namespace": "<MIRROR_STORAGE_NAMESPACE>"
+}'
+```
```

Comment:
Example output or format

Reply:
I don't have such an example (taken from R&D notes..) - feel free to add a better reference in separate PR!